components/tidb_query/src/rpn_expr/types/expr.rs

```diff
@@ -132,6 +132,15 @@ impl RpnExpression {
     pub fn into_inner(self) -> Vec<RpnExpressionNode> {
         self.0
     }
+
+    /// Returns true if the last element of expression is a `Constant` variant.
+    pub fn is_constant(&self) -> bool {
```

Comment:
```suggestion
    pub fn is_last_constant(&self) -> bool {
```
Because it is ambiguas that the expression may not be constant although the last element is a constant.